api/integrations/rss/rss-bot

```diff
@@ -156,10 +173,18 @@ def elide_subject(subject):
 
 def send_zulip(entry, feed_name):
     # type: (Any, str) -> Dict[str, Any]
+    body = entry.summary # type: str
+    if opts.unwrap:
+        body = unwrap_text(body)
+
     content = "**[%s](%s)**\n%s\n%s" % (entry.title,
                                         entry.link,
-                                        strip_tags(entry.summary),
+                                        strip_tags(body),
                                         entry.link) # type: str
+
+    if opts.math:
+        content = content.replace('$', '$$')
```

Comment:
I don't really know what the content is, but what if the content already contains `$$`? Would it become `$$$$`?